src/evict/evict_lru.c

```diff
@@ -448,7 +451,21 @@ __evict_server(WT_SESSION_IMPL *session, bool *did_work)
         return (0);
 
     __wt_epoch(session, &now);
-    if (WT_TIMEDIFF_SEC(now, cache->stuck_time) > WT_MINUTE * 5) {
+
+    /* Calculate how long the cache has been stuck for and the expiry timeout.*/
+    cache_stuck_duration = WT_TIMEDIFF_SEC(now, cache->stuck_time);
+    cache_stuck_timeout = WT_MINUTE * WT_CACHE_STUCK_TIMEOUT;
+
+#if defined(HAVE_DIAGNOSTIC)
+    /* Enable verbose messaging a short period before the cache is declared stuck
```

Comment:
The comment convention is not as per WiredTiger's styling. Can you please check the other multi-line comments and follow them accordingly. Also, it can be shortened to say, "Enable verbose messages a short period before declaring the cache stuck."